README.md

```diff
@@ -1,61 +1,147 @@
+## Configuring Shunting Yard
 
-    sudo restart hive-hcatalog-server
+The examples below all demonstrate configuration using YAML and provide fragments covering the most common use cases that should be useful as a basis for building your own configuration. A full configuration reference is provided in the following sections.
 
-### How to install the receiver - TARGET
+### Configuring source, replica and SQS queue
 
-On the target cluster, download and install the latests version of Circus Train and set the `CIRCUS_TRAIN_HOME` environment variable:
+The YAML fragment below shows some common options for setting up the base source (where data is coming from), replica (where data is going to) and the SQS queue to read hive events from.
 
-    export CIRCUS_TRAIN_HOME=/home/hadoop/circus-train-<circus-train-version>
+    source-catalog:
+      name: source-cluster
+      hive-metastore-uris: thrift://emr-master-1.compute.amazonaws.com:9083
+    replica-catalog:
+      name: replica-cluster
+      hive-metastore-uris: thrift://emr-master-2.compute.amazonaws.com:9083
+    event-receiver:
+      configuration-properties:
+        com.hotels.shunting.yard.event.receiver.sqs.queue: https://sqs.us-west-2.amazonaws.com/123456789/sqs-queue
+        com.hotels.shunting.yard.event.receiver.sqs.wait.time.seconds: 20
+    source-table-filter:
+      table-names:
+        - ...
+    table-replications:
+      ...
 
-Export the location of the Hive libraries:
+### Selecting tables to monitor
 
-    export HIVE_LIB=/usr/lib/hive/lib/
-    export HCAT_LIB=/usr/lib/hive-hcatalog/share/hcatalog/
+The YAML fragment below shows how to select the tables to be monitored by Shunting Yard.
 
-Copy the file _shunting-yard-binary/target/shunting-yard-binary-\<version\>-bin.tgz_ and unzip it, then set the `SHUNTING_YARD_HOME` environment variable:
+    source-table-filter:
+      table-names:
+        - test_database.test_table_1
+        - test_database.test_table_2
 
-    export SHUNTING_YARD_HOME=/home/hadoop/shuting-yard-<version>
+### Specifying target database & table names
 
-Create a `.yml` file under `$SHUNTING_YARD_HOME/conf/` with the correct settings to talk to the messaging infrastructure. You can find guidelines in the file _shunting-yard-minimal.yml.template_. Once this is done you are all set up to run the service:
+The YAML fragments below shows some common options for specifying the target database and table name for the selected tables.
 
-    $SHUNTING_YARD_HOME/bin/replicator.sh --config=$SHUNTING_YARD_HOME/conf/<my-config>.yml
-    
-### Graphite Metrics
+#### Specify both target database and table name
+
+    table-replications:
+      - source-table:
+          database-name: source_database
+          table-name: test_table
+        replica-table:
+          database-name: replica_database
+          table-name: test_table_1    
+          
+#### Only Change the target database but the table name remains same as source
```

Comment:
"change" with lowercase "c"

Reply:
And maybe it would be good to rephrase this as "Change only the target database while keeping the same source table name" to keep the same tone of the writing (as an instruction)

Reply:
done.

Reply:
what about the second half?

Reply:
We talked about this and I'm ok with what's in the document now :)